src/main/java/com/superdevs/HealthOMeter/entity/Contact.java

```diff
@@ -0,0 +1,232 @@
+package com.superdevs.HealthOMeter.entity;
+
+import javax.validation.constraints.NotBlank;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity(name = "contacts")
+public class Contact {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BioMetricData> bioMetricData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BioMetricHistoricalData> bioMetricHistoricalData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<BMIData> bmiData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<WHRData> whrData;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contacts")
+    private Set<RFMData> rfmData;
+
+    @ManyToMany
```

Comment:
No Many@many Relations here. Contact is @manyToOne , and any History related to contact is @oneToMany. No Many@many at all in our Data model